### PR TITLE
[HTML5] - Accessibility Contrast & Semantic Fixes 

### PR DESCRIPTION
**Pull request by KDSBrowne**
Fixes color contrast and aria semantic issues in the client.

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/button/styles.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@ $btn-default-color: $color-gray;
 $btn-default-bg: $color-white;
 $btn-default-border: $color-white;
 
-$btn-primary-color: $color-white;
+$btn-primary-color: #fff;
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/button/styles.scss]: `$color-white` inst `#fff`? :man_facepalming:

File: bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ export default class DropdownListItem extends Component {
         {
           description ?
           (<span id={this.descID} key="describedby" hidden>{description}</span>)
-          : null
+          : (<span id={this.descID} key="describedby" hidden></span>)
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx]: You could just remove the if if you will add an empty element.

`<span id={this.descID} key="describedby" hidden>{description}</span>`

Comment (OZhurbenko) [inline bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx]: Won't it display "undefined" as a string in the case when description doesn't exist?
Probably it should be:
`<span id={this.descID} key="describedby" hidden>{description ? description : null}</span>`

Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx]: @OZhurbenko undefined variables are rendered the same way as null or empty.

Comment (OZhurbenko) [inline bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx]: That's true, tested it. 👍

File: bigbluebutton-html5/imports/ui/components/button/styles.scss (hunk 1 above)
Comment (KDSBrowne) [inline bigbluebutton-html5/imports/ui/components/button/styles.scss]: $color-white is set to #F3F6F9

File: bigbluebutton-html5/imports/ui/components/user-avatar/styles.scss [hunk 3]
```diff
@@ -4,14 +4,18 @@
  * ==========
  */
 $user-avatar-border: $color-gray-light;
-$user-avatar-text: $user-avatar-border;
+$user-avatar-text: #fff;
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/user-avatar/styles.scss]: `$color-white`

File: bigbluebutton-html5/imports/ui/components/modal/fullscreen/styles.scss [hunk 4]
```diff
@@ -48,4 +48,6 @@
 }
 
 .confirm {
+  color: #fff !important;
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/modal/fullscreen/styles.scss]: `$color-white`

File: bigbluebutton-html5/imports/ui/components/dropdown/list/title/component.jsx [hunk 5]
```diff
@@ -8,14 +8,19 @@ const propTypes = {
 
 export default class DropdownListTitle extends Component {
 
+  constructor(props) {
+    super(props);
+    this.labelID = _.uniqueId("labelContext-");;
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/dropdown/list/title/component.jsx]: lint

**Pull request merged**